assets/ASSETS/Assets.h

```diff
@@ -4,9 +4,9 @@
 #include <unordered_map>
 #include <memory>
 
-#include <SDL2/SDL.h>
-#include <SDL2/SDL_image.h>
-#include <SDL2/SDL_mixer.h>
+#include <SDL.h>
```

Comment:
I hit a problem with these includes (https://stackoverflow.com/questions/37817717/sdl2-library-not-found-with-make-but-with-cmake)
and had to change them.
I don't know how compatible it is with your setup or other OSs, but it sounds like it's the way to do it?

```
# Note that the header path has changed from SDL2/SDL.h to just SDL.h
# This needed to change because "proper" SDL2 convention
# is #include "SDL.h", not <SDL2/SDL.h>. This is done for portability
# reasons because not all systems place things in SDL2/ (see FreeBSD).
```